cava/samples/cudart/cudart.cpp

```diff
@@ -137,14 +139,14 @@ ava_utility int __helper_cubin_num(void **cubin_handle) {
 ava_utility void __helper_dump_fatbin(void *fatCubin,
                                     GHashTable **fatbin_funcs,
                                     int *num_funcs) {
-    struct fatbin_wrapper *wp = fatCubin;
+    struct fatbin_wrapper *wp = (struct fatbin_wrapper *)fatCubin;
```

Comment:
static_cast is better.

Reply:
`static_cast<type>(value)` syntax is not parse-able by CAvA yet. Leave it as TODO.